unitTests/ui/MLFacet/MLFacetBreadcrumbsTest.ts

```diff
@@ -0,0 +1,95 @@
+import { MLFacetBreadcrumbs } from '../../../src/ui/MLFacet/MLFacetBreadcrumbs';
+import { $$ } from '../../../src/Core';
+import { MLFacetTestUtils } from './MLFacetTestUtils';
+import { MLFacet, IMLFacetOptions } from '../../../src/ui/MLFacet/MLFacet';
+import { IMLFacetValue } from '../../../src/ui/MLFacet/MLFacetValues/MLFacetValue';
+import { FacetValueState } from '../../../src/rest/Facet/FacetValueState';
+
+export function MLFacetBreadcrumbsTest() {
```

Comment:
I'd also add some basic tests on the top level MLFacet component to verify that it handles the "populateBreadcrumb" event properly + that it respects all the options related to breadcrumb ( eg: enable/disable breadcrumbs, number of values in breadcrumb, etc.)